docker-compose.yml

```diff
@@ -7,19 +7,21 @@ services:
       - db-data:/var/lib/postgresql/data
     environment:
       - POSTGRES_PASSWORD=postgres
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
 
   rallly:
     build:
       context: .
-      args:
-        - DATABASE_URL=postgres://postgres:postgres@rallly_db:5432/db?pgbouncer=true
     restart: always
     depends_on:
-      - rallly_db
+      rallly_db:
+        condition: service_healthy
     ports:
       - 3000:3000
-    environment:
-      - DATABASE_URL=postgres://postgres:postgres@rallly_db:5432/db
     env_file:
```

Comment:
Thanks @LarsHaalck. I think these two lines should stay to avoid having to manually add this to the `.env` file.

Reply:
Okay no problem.

I think the default db, postgres creates inside the container is the same as the `POSTGRES_USER` variable which defaults to `postgres`. So we either have to change the `DATABASE_URL` or add `POSTGRES_DB=db` to the db container. What do you think?

Reply:
I don't think setting `POSTGRES_DB` is enough. `DATABASE_URL` is required to generate the prisma client. It won't work with any other variables.

See here: https://github.com/lukevella/rallly/blob/main/prisma/schema.prisma#L3

Reply:
Ah I see. I wonder how this never caused an issue before because `POSTGRES_DB` was never set and the connection string was always `postgres://postgres:postgres@rallly_db:5432/postgres`. 

Anyway, setting `POSTRES_DB=db` for rallly_db makes sense, but how would the prisma client get the database connection string without setting `DATABASE_URL` as well?

Reply:
I would leave the `DATABASE_URL` and also set `POSTRES_DB=db`.

Interestingly, it also works for me but I get the following error messages when not setting `POSTRES_DB`:
```
rallly-rallly_db-1  | 2022-10-11 09:09:48.915 UTC [75] FATAL:  database "db" does not exist
rallly-rallly_db-1  | 2022-10-11 09:09:48.945 UTC [76] FATAL:  database "db" does not exist
rallly-rallly-1     |
rallly-rallly-1     | PostgreSQL database db created at rallly_db:
```

So it seems, the db is just created when it doesn't exist. I think it would be cleaner to set both environment variables. Let me put the changes in another commit and then you can check again.